build/rubberband-1.8.1/rubberband-library.vcxproj

```diff
@@ -87,7 +87,7 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
       <Optimization>Disabled</Optimization>
-      <PreprocessorDefinitions>__MSVC__;WIN32;HAVE_FFTW3;_DEBUG;_LIB;NOMINMAX;_USE_MATH_DEFINES;USE_KISSFFT;USE_SPEEX;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>__MSVC__;WIN32;$(TypeOfFftwLinking);HAVE_FFTW3;_DEBUG;_LIB;NOMINMAX;_USE_MATH_DEFINES;USE_KISSFFT;USE_SPEEX;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Hm, sidenote -- I just noticed we should probably define FFTW_SINGLE_ONLY since we only use rubberband for floats. We should only need fftw3f in Mixxx, I'm not sure what else is linking against fftw (maybe vamp?).